assets/components/src/web-preview/index.js

```diff
@@ -48,15 +48,6 @@ class WebPreview extends Component {
 		}
 	}
 
-	/**
```

Comment:
FYI @adekbadek I had to remove the `componentWillUnmount` after finding that exiting a view with multiple `WebPreview` components caused a crash. If there are multiple instances of the component, the first one to unmount will remove the DOM element leaving the others with a reference to a non-existent element. When the other unmount this throws an error. Also,  we wouldn't want a situation where one `WebPreview` unmounts and removes the DOM element that is still in being used by another. All this means is we may be left  with a single  empty div sitting around, which isn't really so bad...

Reply:
Yeah, I guess in this case it's ok to skip the cleanup